release/gcb/local.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Why is this under gcb directory? Does it depend on gcb?

Reply:
Because the rest of the scripts are here, that's the only reason.